src/liquid/components/ld-input/ld-input.tsx

```diff
@@ -73,6 +73,9 @@ export class LdInput implements InnerFocusable, ClonesAttributes {
   /** Set this property to `true` in order to mark the field visually as invalid. */
   @Prop() invalid?: boolean
 
+  /** Tab index of the input. */
+  @Prop() ldTabindex: number | undefined
```

Comment:
```suggestion
  @Prop() ldTabindex?: number
```

Reply:
Doing it like this, I'd have to change the "InnerFocusable" interface, as well. This would result in TS no longer showing an error, if a developer does not define this prop in a component implementing this interface. I would leave it as is. Do you agree?